frameworks/Go/hertz/hello.go

```diff
@@ -66,122 +72,208 @@ func parseQueries(c context.Context, ctx *app.RequestContext) int {
 }
 
 // / Test 1: JSON serialization
-func json(c context.Context, ctx *app.RequestContext) {
-	ctx.JSON(200, utils.H{"message": "Hello, World!"})
+func jsonhandler(c context.Context, ctx *app.RequestContext) {
+	m := AcquireJSON()
+	m.Message = helloworld
+	ctx.JSON(200, &m)
+	ReleaseJSON(m)
 }
 
 // / Test 2: Single database query
-func db(c context.Context, ctx *app.RequestContext) {
-	var world World
-	err := worldStatement.QueryRow(rand.Intn(worldRowCount)+1).Scan(&world.Id, &world.RandomNumber)
-	if err != nil {
-		ctx.AbortWithError(500, err)
-		return
-	}
+func dbHandler(c context.Context, ctx *app.RequestContext) {
+	world := AcquireWorld()
+	db.QueryRow(context.Background(), worldSelect, RandomWorld()).Scan(&world.Id, &world.RandomNumber)
 	ctx.JSON(200, &world)
+	ReleaseWorld(world)
 }
 
 // / Test 3: Multiple database queries
 func dbs(c context.Context, ctx *app.RequestContext) {
-	numQueries := parseQueries(c, ctx)
-
-	worlds := make([]World, numQueries)
-	for i := 0; i < numQueries; i++ {
-		err := worldStatement.QueryRow(rand.Intn(worldRowCount)+1).Scan(&worlds[i].Id, &worlds[i].RandomNumber)
-		if err != nil {
-			ctx.AbortWithError(500, err)
-			return
-		}
+	n := parseQueries(c, ctx)
+	worlds := AcquireWorlds()[:n]
+	for i := 0; i < n; i++ {
+		w := &worlds[i]
+		db.QueryRow(context.Background(), worldSelect, RandomWorld()).Scan(&w.Id, &w.RandomNumber)
 	}
 	ctx.JSON(200, &worlds)
+	ReleaseWorlds(worlds)
 }
 
 // / Test 4: Fortunes
 func fortunes(c context.Context, ctx *app.RequestContext) {
-	rows, err := fortuneStatement.Query()
-	if err != nil {
-		ctx.AbortWithError(500, err)
-		return
-	}
-
+	rows, _ := db.Query(context.Background(), fortuneSelect)
 	fortunes := make(Fortunes, 0, 16)
```

Comment:
Not sure how this was missed before, but you cannot allocate memory with prior knowledge of how many fortunes there are.

https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#fortunes

> v. The list data structure must be a dynamic-size or equivalent and should not be dimensioned using foreknowledge of the row-count of the database table.

Reply:
This code mimics the code from fiber
```go
fortunes := make([]templates.Fortune, 0, 16)
```

Reply:
@nbrady-techempower 
I've also fixed all the early memory allocation issues in the go framework, so please merge as soon as possible if you can, it's very important to me.